ClientApp/app/reducers/error-message.reducer.ts

```diff
@@ -0,0 +1,34 @@
+﻿import { Action } from '@ngrx/store';
+
+import { ErrorMessage } from '../models/error-message';
+import { ErrorActions } from '../actions/error.actions';
+
+export type ErrorMessageState = ErrorMessage;
+
+const initialState: ErrorMessageState = {
+    statusCode: 0,
+    message: ''
+};
+
+export default function (state = initialState, action: Action): ErrorMessageState {
+    switch (action.type) {
+        case ErrorActions.CATCH_ERROR: {
+            let obj: ErrorMessageState = Object.assign({}, state);
+            obj.message = action.payload.error.message;
+            obj.statusCode = action.payload.code;
+            if (action.payload.error.entity) {
```

Comment:
I'd say that for end user this information about entity type and id is useless most of the time. Just message is enough